src/cc/frontends/clang/b_frontend_action.cc

```diff
@@ -185,7 +186,7 @@ bool ProbeVisitor::VisitBinaryOperator(BinaryOperator *E) {
   if (ProbeChecker(E->getRHS(), ptregs_).is_transitive()) {
     ProbeSetter setter(&ptregs_);
     setter.TraverseStmt(E->getLHS());
-  } else if (E->isAssignmentOp() && E->getRHS()->getStmtClass() == Stmt::CallExprClass) {
+  } else if (E->getRHS()->getStmtClass() == Stmt::CallExprClass) {
```

Comment:
Could you explain why you remove `E->isAssignmentOp()` here?

Reply:
Because we're sure `E->isAssignmentOp()` is true when we reach this point (see 6 lines above). I should have commented on that. Sorry.

Reply:
Oh, yes. It is an apparent from the whole source code. Sorry I did not check in the context of the whole file.